README_CN.md

```diff
@@ -0,0 +1,621 @@
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>R</kbd></td>
+        </tr>
+        <tr>
+            <td>Dispose Current SDK Configuration Editor Server Process</td>
+            <td>若先前执行过 SDK Configuration Editor 命令，则后台将保留缓存进程，以便下次更快打开编辑器。此命令将清理此类缓存进程。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Doctor Command</td>
+            <td>诊断扩展设置及扩展日志，提供故障排除报告。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Troubleshoot Form</td>
+            <td>启动 UI，以便用户发送故障排除报告，报告中需包含重现问题的步骤。同时系统将诊断扩展设置及扩展日志，并将信息发送到遥测后端。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Run ESP-IDF-SBOM vulnerability check</td>
+            <td>为使用 ESP-IDF 开发框架生成的应用程序创建 SPDX 格式的软件物料清单（SBOM）文件。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Save Default SDKCONFIG file (save-defconfig)</td>
+            <td>使用当前项目的 sdkconfig 文件，生成 sdkconfig.defaults 文件。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Show Ninja Build Summary</td>
+            <td>运行 Chromium ninja-build-summary.py。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Search in documentation...</td>
+            <td>从源代码文件中选择文本，并在 ESP-IDF 文档中进行搜索，搜索结果将显示在 VSCode ESP-IDF Explorer 选项卡中。</td>
+            <td><kbd>⌘</kbd> <kbd>I</kbd> <kbd>Q</kbd></td>
+            <td><kbd>Ctrl</kbd> <kbd>E</kbd> <kbd>Q</kbd></td>
+        </tr>
+        <tr>
+            <td>Search Error Hint</td>
+            <td>输入文本，在 ESP-IDF 提示库中搜索匹配的错误。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td rowspan=2 align="center"> 清理</td>
+            <td>Clear ESP-IDF Search Results</td>
+            <td>清除 Explorer 中 Documentation Search Results 选项卡中的所有搜索结果。</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>Clear Saved ESP-IDF Setups</td>
+            <td>清除扩展中保留的现有 ESP-IDF 设置。</td>
+            <td></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+## 针对 tasks.json 和 launch.json 的命令
+
+扩展中实现了一些实用工具命令，可以在 tasks.json 和 launch.json 文件中按照如下方式使用：
+
+```json
+"miDebuggerPath": "${command:espIdf.getToolchainGdb}"
+```
+
+- `espIdf.getExtensionPath`：获取已安装位置的绝对路径。
+- `espIdf.getOpenOcdScriptValue`：返回从 ESP-IDF 工具路径、`idf.customExtraVars` 或系统 OPENOCD_SCRIPTS 环境变量中计算出的 OPENOCD_SCRIPTS 的值。
+- `espIdf.getOpenOcdConfig`：以字符串形式返回 openOCD 配置文件。例如 `-f interface/ftdi/esp32_devkitj_v1.cfg -f board/esp32-wrover.cfg`。
+- `espIdf.getProjectName`：从当前工作区文件夹的 `build/project_description.json` 文件中提取项目名称。
+- `espIdf.getToolchainGcc`：根据 sdkconfig 文件中指定的 IDF_TARGET，该命令将返回相应 GCC 工具链的绝对路径。
+- `espIdf.getToolchainGdb`：根据 sdkconfig 文件中指定的 IDF_TARGET，该命令将返回相应 GDB 工具链的绝对路径。
+
+在[调试](./docs/DEBUGGING.md)文档中查看示例。
+
+## tasks.json 中可用的任务
```

Comment:
```suggestion
## tasks.json 模版

Reply:
I think it might be better to keep the original title "tasks.json 中可用的任务". "tasks.json 模版" seems less clear, as the following content focuses more on the specific pre-defined tasks rather than the structure of this "template." What do you think? @April-Yjj

Reply:
Sounds reasonable! My concern was that the title can help readers find templates when they want templates. Task sounds general from this perspective. What do you think of  **task.json 模板任务**?